ourjapanlife/findadoc-web
# test: add cypress testing for pages

Part of #502 

## 🔧 What changed

This PR focuses on improving code quality by implementing testing for pages. Focus was put on testing pages rather than individual components, to cover how the layout appears when everything is rendered together.

## 🧪 Testing instructions
1. Check that CI is passing
2. Start the frontend server with `yarn dev` and run Cypress locally with `yarn cypress open`
3. Confirm that everything in the preview still works
